package.json

```diff
@@ -30,9 +41,11 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
+    "bcrypt": "^5.1.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "express": "^4.18.2",
+    "migrations": "^1.3.0",
```

Comment:
Why you'r using this package?

Reply:
I mean where did you this library?

Reply:
Yes, I thought I had to install it to do migration.